modules/deploying-models-on-the-single-model-serving-platform.adoc

```diff
@@ -31,14 +31,18 @@ endif::[]
 ifdef::cloud-service[]
 * (Knative Serverless deployments only) To enable token authentication and external model routes for deployed models, you have added Authorino as an authorization provider. For more information, see link:{rhoaidocshome}{default-format-url}/installing_and_uninstalling_{url-productname-short}/installing-the-single-model-serving-platform_component-install#adding-an-authorization-provider_component-install[Adding an authorization provider for the single-model serving platform].
 endif::[]
-ifdef::upstream[]
+ifndef::upstream[]
 * (Knative Serverless deployments only) To enable token authentication and external model routes for deployed models, you have added Authorino as an authorization provider.
 endif::[]
 * You have created a data science project.
-* You have access to S3-compatible object storage, URI-based repository, or OCI-compliant registry and have added a connection to your data science project. For more information about adding a connection, see link:{rhoaidocshome}{default-format-url}/working_on_data_science_projects/using-connections_projects#adding-a-connection-to-your-data-science-project_projects[Adding a connection to your data science project].
 ifndef::upstream[]
+* You have access to S3-compatible object storage, URI-based repository, OCI-compliant registry or persistent volume claim (PVC) and have added a connection to your data science project. For more information about adding a connection, see link:{rhoaidocshome}{default-format-url}/working_on_data_science_projects/using-connections_projects#adding-a-connection-to-your-data-science-project_projects[Adding a connection to your data science project].
```

Comment:
Sorry - did not catch during my first review:
I think you need to add some "a"s and "an":
You have access to S3-compatible object storage, **a** URI-based repository, **an** OCI-compliant registry or **a** persistent volume claim (PVC)